manageiq-operator/pkg/controller/manageiq/manageiq_controller.go

```diff
@@ -101,6 +106,26 @@ func (r *ReconcileManageIQ) Reconcile(request reconcile.Request) (reconcile.Resu
 	reqLogger := logger.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
 	reqLogger.Info("Reconciling ManageIQ")
 
+	if strings.HasPrefix(request.Name, "httpd-auth-config-secret-") {
+		// Mapped secret name coming as httpd-auth-config-secret-<miq_instance_name>-<secret_name>
+		rString := strings.TrimPrefix(request.Name, "httpd-auth-config-secret-")
+		nSlices := strings.SplitN(rString, "-", 2)
+		// Fetch the ManageIQ instance
+		miqInstance := &miqv1alpha1.ManageIQ{}
+		err := r.client.Get(context.TODO(), types.NamespacedName{
+			Namespace: request.Namespace,
+			Name:      nSlices[0],
```

Comment:
What do you think of this?
```suggestion
		nSlices := strings.SplitN(rString, "-", 6)
		// Fetch the ManageIQ instance
		miqInstance := &miqv1alpha1.ManageIQ{}
		err := r.client.Get(context.TODO(), types.NamespacedName{
			Namespace: request.Namespace,
			Name:      nSlices[4],
```